--- a/cmake/android/theora/state.c
+++ b/cmake/android/theora/state.c
@@ -16,6 +16,7 @@
 
 #include <stdlib.h>
 #include <string.h>
+#include <stdint.h>
 #include "state.h"
 #if defined(OC_DUMP_IMAGES)
 # include <stdio.h>
@@ -388,7 +389,7 @@
         /*Otherwise, check to see if it straddles the border.*/
         else if(x<crop_x0&&crop_x0<x+8||x<crop_xf&&crop_xf<x+8||
          y<crop_y0&&crop_y0<y+8||y<crop_yf&&crop_yf<y+8){
-          ogg_uint64_t mask;
+          uint64_t mask;
           int         npixels;
           int         i;
           mask=npixels=0;
@@ -396,7 +397,7 @@
             int j;
             for(j=0;j<8;j++){
               if(x+j>=crop_x0&&x+j<crop_xf&&y+i>=crop_y0&&y+i<crop_yf){
-                mask|=(ogg_uint64_t)1<<(i<<3|j);
+                mask|=(uint64_t)1<<(i<<3|j);
                 npixels++;
               }
             }
